lib/sda/node.go

```diff
@@ -255,6 +268,13 @@ func (n *Node) Shutdown() error {
 	return nil
 }
 
+// Close shuts down the go-routine and calls the protocolInstance-shutdown
+func (n *Node) Close() error {
+	dbg.Lvl3("Closing node", n.Myself())
```

Comment:
`MySelf()` ? 
I'm not sure this is a good idea, `Name()` is very explicit and the callers knows what to expect: a string. With `Myself()`, we don't know what to expect, is it a struct? a string? For some people coming from C++ / Java, `self` has another meaning.